web/ts/TUMLiveVjs.ts

```diff
@@ -584,6 +584,14 @@ export function jumpTo(hours: number, minutes: number, seconds: number) {
     }
 }
 
+export function jumpToMs(ms: number) {
```

Comment:
Could you maybe merge the exisiting `jumpTo` function with this one by 
changing the parameter to an object? e.g.:

```typescript 
export function jumpTo(setting: object) {
  for (let j = 0; j < players.length; j++) {
         players[j].ready(() => {
           if ('ms' in object) { ... }
           else { ... }
       }
    }
}
```

Reply:
Yup, good catch :+1: